gulpfile.js

```diff
@@ -54,7 +54,11 @@ function getPlatforms(includeChromeOs) {
 
             break;
         case 'linux':
-            platforms.push('linux64');
+            if (os.arch()==='ia32') {
```

Comment:
See my previous comment. `linux` should build `linux64` **unless** `--linux32` is explicitly specified.